### ISSUE TITLE
is it possible to use already binded jni for android?

### ISSUE THREAD

**Issue by eix128** [bytedeco/javacpp-presets#816]
Hi , i have seen Cheat Engine can be used on android by jni.
is it possible to make this library use with java ?
https://github.com/cheat-engine/cheat-engine/tree/master/Cheat%20Engine/jni

There is also
https://github.com/fenix01/cheatengine-library

**Comment by saudet** [comment]
Yes, it's possible. The official Java APIs of, for example, OpenCV, MXNet, and TensorFlow are already being bundled in the presets.
